Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -230,6 +230,18 @@ CookieStore::~CookieStore()
     m_mainThreadBridge->detach();
 }
 
+static bool containsInvalidCharacters(const String& str)
+{
+    for (unsigned i = 0; i < str.length(); i++) {
+        auto currentChar = str[i];
+        if (currentChar == 59 || currentChar == 127)
```

Comment:
We usually use hexadecimal to match the spec, e.g. `0x003B`, instead of decimal values.

Reply:
Fixed with a comment to the spec so the numbers seem less random.